specs/spec-1-kudos.md

```diff
@@ -1,107 +1,269 @@
----
-#: 1
-Status: Review
-Category: NEAR Social
-Created: 2023-04-05
-Replaces: --
-Requires: --
----
+<!-- Objective. In a few sentences, describe the key system objectives. Tweet length executive summary. -->
+# RFP 2: Kudos Widget
```

Comment:
Is this RFP or spec?
The file is in spec but the title suggest RFP.

Reply:
I say Spec. The way I imagined the Kudos process was that we would include all the detail we could in the Spec, put it up for RFP, and then fill in further Spec details depending on the solution provided by the RFP winner.

Reply:
Yes, I also think it should be a spec.

Reply:
`improvement-proposals` was missing, I've synced your branch so now you should see the proper errors.
LMK if you need a hand going through it (ping me in telegram)

Reply:
oh, it seams now there are no more errors, looks good :)